src/patternfly/components/Tooltip/styles.scss

```diff
@@ -0,0 +1,131 @@
+@import "../../patternfly-utilities";
+
+.pf-c-overlay {
+  --pf-c-overlay__container--ZIndex: var(--pf-global--ZIndex--xl);
+
+  position: relative;
+  display: inline-block;
+
+  &__container {
+    position: absolute;
+    z-index: var(--pf-c-overlay__container--ZIndex);
+    display: flex;
+    width: 100vw;
+
+    // necessary due to setting width
+    // this fixes pointer events using a hack
+    // unsure about browser compatibility of this hack
```

Comment:
This comment needs more clarity.

Setting `width: 100vw` is necessary since the `__container` is absolutely positioned and we want to allow the content enough space to work in and have its layout flow. However, the `__container` width creates an invisible block where a user cannot click through. Therefore the `__container` is given `pointer-events: none` to allow clicking elements underneath. This negatively impacts the content and we need to enable pointer events on all children of the `__container` to allow the content to be interactive.